src/Azure.Functions.Cli/Actions/AzureActions/LoginAction.cs

```diff
@@ -1,23 +1,63 @@
 ﻿using System;
-using System.Linq;
+using System.Text;
 using System.Threading.Tasks;
-using Colors.Net;
 using Azure.Functions.Cli.Arm;
 using Azure.Functions.Cli.Interfaces;
+using Fclp;
 
 namespace Azure.Functions.Cli.Actions.AzureActions
 {
     [Action(Name = "login", Context = Context.Azure, HelpText = "Log in to an Azure account")]
     class LoginAction : BaseAzureAccountAction
     {
+        private string _username = string.Empty;
+        private string _password = string.Empty;
+
         public LoginAction(IArmManager armManager, ISettings settings)
             : base(armManager, settings)
         { }
 
+        public override ICommandLineParserResult ParseArgs(string[] args)
+        {
+            Parser.Setup<string>('u')
+                .WithDescription("username")
+                .Callback(username => _username = username);
+
+            Parser.Setup<string>('w')
+                .WithDescription("password")
+                .Callback(password => _password = password);
```

Comment:
Can you add a long option for these as well for consistency with other flags?

Maybe the description can be `Username to use for non-interactive login. Note that accounts with 2 factor-auth require interactive login. Default: interactive` and `Password to use for non-interactive login only in conjunction with -u. Default: prompt`